presto-docs/src/main/sphinx/sql/select.rst

```diff
@@ -781,6 +781,65 @@ The following query will fail with the error ``Column 'name' is ambiguous``::
     FROM nation
     CROSS JOIN region;
 
+
+USING
+^^^^^^^^^^^^^^^^^^^^^^^
+Making a join with `USING` helps write shorter code. For example, instead of the following code:
+
+.. code-block:: none 
+
+    SELECT
+        *
+    FROM t1 
+    JOIN t2
+    ON t1.key_1 = t2.key_1 AND t1.key_2 = t2.key_2
+
+You can write:
+
+.. code-block:: none 
+
+    SELECT
+        *
+    FROM t1
+    JOIN t2
+    USING (key_1, key_2)
+
+When using `USING` the column names of `key_1` and `key_2` are no longer part of the original tables (t1 and t2). Thus, using * will NOT add two duplicate key columns from t1 and t2. If we want to get the explicit columns from each of the tables, we'd call them directly without the table prefix (i.e.: key_1), and using `t1.key_1` will return an error. 
```

Comment:
This part is confusing to me. Can you change this to something like the below? (my text probably needs to be edited for formatting since I wrote it up in this comment box.)
```
The output of ``JOIN USING`` will be one copy of the join key columns (key_1 and key_2 in the example 
above) followed by the remaining columns in t1 and then the remaining columns in t2. Note that the join 
keys are not included in the list of columns from the origin tables for the purpose of referencing them in the 
query. You cannot qualify them with a table prefix and if your run ``SELECT t1.*``, the join columns are not 
included in the output.

The following two queries are equivalent:
.. code-block:: none 
SELECT *
    FROM (
        VALUES
            (1, 3, 10),
            (2, 4, 20)
    ) AS table_1 (key_1, key_2, y1)
    LEFT JOIN (
        VALUES
            (1, 3, 100),
            (2, 4, 200)
    ) AS table_2 (key_1, key_2, y2) 
    USING (key_1, key_2)

.. code-block::none
     SELECT key_1, key_2, table_1.*, table_2.*
    FROM (
        VALUES
            (1, 3, 10),
            (2, 4, 20)
    ) AS table_1 (key_1, key_2, y1)
    LEFT JOIN (
        VALUES
            (1, 3, 100),
            (2, 4, 200)
    ) AS table_2 (key_1, key_2, y2) 
    USING (key_1, key_2)

And produce the output:
.. code-block:: none

 key_1 | key_2 | y1 | y2  
-------+-------+----+-----
     1 |     3 | 10 | 100 
     2 |     4 | 20 | 200 
(2 rows)
```